roles/kubernetes/node/templates/kubelet.rkt.service.j2

```diff
@@ -32,6 +32,7 @@ ExecStart=/usr/bin/rkt run \
         --volume etc-cni,kind=host,source=/etc/cni,readOnly=true \
         --volume opt-cni,kind=host,source=/opt/cni,readOnly=true \
         --volume var-lib-cni,kind=host,source=/var/lib/cni,readOnly=false \
+        --volume local-volume-base-dir,target {{ local_volume_base_dir }},readOnly=false \
```

Comment:
I think we'll want `,recursive=true` added to this volume

Reply:
Yeah we want to mount all host submounts